bin/pycbc_optimize_snr

```diff
@@ -196,18 +206,25 @@ for ifo in args.data_files.keys():
 
 fp = h5py.File(args.params_file, 'r')
 ifos = list(data.keys())
-original_gid = fp['gid'][()]
+try:
```

Comment:
Since the lack of `gid` seems an "expected" occurrence (I guess it happens if the search was told not to upload to GraceDB?) I would use an explicit check here (`if 'gid' in fp`) instead of a try-except.